hello-world/src/index.html

```diff
@@ -0,0 +1,32 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <title>React BootCamp</title>
+    <!-- Core React Code (shared file) -->
+    <script
```

Comment:
reviewed. code is fine. script tags for react and react dom not needed when already installed react as a library. But its fine if you kept for your reference for assignment 1.

Reply:
yes kept it for reference and created a new folder for assignment 2.